kvmagent/kvmagent/plugins/qga_zwatch.py

```
@@ -298,6 +305,9 @@ def get_state(domain):
                 if not config:
                     logger.debug("read /usr/local/zstack/guesttools failed")
                     return qga_status
+                time_protocol = qga.guest_exec_bash_no_exitcode(
+                    "grep -v '^[[:space:]]*#' /etc/chrony.conf | grep '/dev/ptp0' > /dev/null && echo 'ptp' || echo 'ntp'").strip()
+                qga_status.timeProtocol = time_protocol
             except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)